ksql-engine/src/main/java/io/confluent/ksql/KsqlContext.java

```diff
@@ -112,6 +112,10 @@ public void sql(String sql) throws Exception {
     return ksqlEngine.getLiveQueries();
   }
 
+  public KsqlEngine getKsqlEngine() {
```

Comment:
Do we want to expose this? i.e., it seems it is only required for the purpose of the test.

Reply:
As I said below, we need a reference to the `ksqlEngine` to submit non-persistent queries. The integration test is in a separate package, so it can't be package-private. Open to suggestions for how to resolve this. One option is to enable the `ksqlContext.sql` method to run non-persistent queries. But I am not sure if that makes sense from an API perspective.

Reply:
I think we need another way where we don't expose the `KsqlEngine` from an API PoV it is a big increase in the surface area

Reply:
@bluemonk3y @hjafarpour thoughts on this? i.e, it seems that the `KsqlContext` is pretty limited. Is this ok? Should we make it a bit more useable, so we can easily do what @apurvam is trying to achieve without having to get a handle to the `ksqlEngine`?

Reply:
I agree, KsqlContext should not expose the engine. 
@apurvam maybe you can use KsqlEngine directly in your test.

Reply:
I updated the code to use `KSQLEngine` directly.